UDV-Core/src/Modules/ConsultDoc/DocumentBrowser.js

```diff
@@ -191,7 +191,10 @@ export function DocumentBrowser(browserContainer, documentController) {
 
       if (this.currentDoc != null & this.documentsExist == true)
       {
-        this.currentMetadata = this.currentDoc.metadata;
+
+        $('#docDeleteButton').show();
+        $('#docUpdateButton').show();
```

Comment:
Why DocumentBrowser knows about Delete and update buttons ?

Reply:
this is used is the case when no document is found when filters are set. Delete and Update buttons are hidden. 
The two lines above are used to show the buttons if documents are found

Reply:
This is now handled through a function